typings/objection/index.d.ts

```diff
@@ -29,8 +29,8 @@ declare namespace Objection {
   const compose: Compose;
   const mixin: Mixin;
 
-  const snakeCaseMappers: () => ColumnNameMappers;
-  const knexSnakeCaseMappers: () => KnexMappers;
+  const snakeCaseMappers: (opt?: any) => ColumnNameMappers;
+  const knexSnakeCaseMappers: (opt?: any) => KnexMappers;
```

Comment:
The options should be typed. You should create an interface like `SnakeCaseMapperOptions` and use that as the argument.

Reply:
Just done.